docs/account/emails.md

```diff
@@ -2,12 +2,44 @@
 
 To manage the email addresses associated with your account and your email notifications, click on your avatar on the top right-hand corner and open **Your account**, page **Emails**.
 
-## Updating your email addresses
+![Email settings](images/emails-notifications.png)
+
+## Updating your email addresses {: id="updating"}
+
+Codacy pulls email addresses from the Git provider associated with your current session. On the **Emails** page, you can verify which email addresses are associated with your Codacy account.
+
+To update the email addresses associated with your Codacy account, do the following:
 
-Codacy pulls email addresses from the Git provider associated with your current session. To update the email addresses displayed on Codacy, update them on your Git provider, then log out and back in to Codacy.
+1.  Make sure you [set your Git email address](#git-config).
+
+1.  Update your email addresses on your Git provider ([GitHub](https://docs.github.com/en/enterprise-cloud@latest/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/adding-an-email-address-to-your-github-account), [Bitbucket](https://support.atlassian.com/bitbucket-cloud/docs/set-email-aliases/), or [GitLab](https://docs.gitlab.com/ee/user/profile/#add-emails-to-your-user-profile)).
+
+1.  Log out and log back in to Codacy.
+
+    !!! tip
+        If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.
 
 !!! note
-    If the updates are still not reflected on Codacy, navigate to the [access management](https://app.codacy.com/account/access-management) page, revoke the relevant Git provider or Google integration, then log out and back in to Codacy using the same provider.
+    **When using GitHub**, Codacy automatically associates all commit email addresses linked to the same GitHub user with a single committer. Thus, developers that never logged in to the Codacy app must also make sure they [set their Git email address](#git-config) and add all their email addresses to their [GitHub account](https://github.com/settings/emails).
```

Comment:
We only do so as commits are made, we don't do it retro-actively. We should make this clear in the docs so that people don't think that emails from a 10k commit gut history will be merged.

Reply:
Nice catch, @lorandszakacs ! 👍 

```suggestion
    When developers commit **from GitHub**, Codacy automatically associates all email addresses linked to their GitHub user with a single Codacy committer. Thus, developers that never logged in to the Codacy app must also make sure they [set their Git email address](#git-config) and add all their email addresses to their [GitHub account](https://github.com/settings/emails).
```